src/data/trips/path.ts

```diff
@@ -60,6 +60,34 @@ export async function updatePath(pathId: number, payload: Partial<PathInput>): P
 			}
 		}
 
+		throw error;
+	}
+}
+
+/**
+ * Obtient le chemin qui va vers l'étape donné
+ * @param stepId identifiant de l'étape de destination
+ * @returns chemin correspondant
+ */
+export async function getPathByStep(stepId: number): Promise<PathOutput> {
```

Comment:
You should rename this function to something like 'getPathToStep' which more closely matches the description and the use case